info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -60,12 +60,16 @@
   private final static String VOTER_INFO_QUERY_URL =
       String.format("https://www.googleapis.com/civicinfo/v2/voterinfo?key=%s",
                     Config.CIVIC_INFO_API_KEY);
+  private final static String REPRENTATIVE_QUERY_URL =
+      String.format("https://www.googleapis.com/civicinfo/v2/representatives?key=%s",
+                    Config.CIVIC_INFO_API_KEY);
   private Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
   private List<String> electionQueryIds = new ArrayList<>();
   // For testing purposes (not to add too much information to the database).
   // Will include all 50 states.
   private List<String> states = Arrays.asList("NY");
   private WebCrawler webCrawler;
+  private String ocdDivisionId;
```

Comment:
This variable is not used across the class. Consider making this variable a local variable within the method that uses it.